### PR TITLE
#21 Ft logout 21

### PR DESCRIPTION
**Pull request by Hodal66**
### What does this PR do?

Create, read, update and delete routes

**Description of Task to be completed?**

As an authenticated user, I want to be able to logout, so that none else can access my account as well

**How should this be manually tested?**

Clone the repo
checkout to branch `ft-logout-21`

_pull the changes by git pull_

run` npm  install `
run `npm run start:dev`

_Go to postman_  

Method: POST http://localhost:3000/api/v1/auth/login.      ===> first login
Method: POST http://localhost:3000/api/v1/auth/logout.   ===>then logout

What are the relevant pivotal tracker stories?
[Trello Board](https://trello.com/c/jX4XhrrS/21-logout-from-account)

Screenshots (if appropriate)
N/A
Screen Shot 2022-06-23 at 12 11 01

Questions:
N/A

### CODE REVIEW

File: src/Controllers/Authentication/login.controller.js [hunk 1]
```diff
@@ -3,51 +3,83 @@
 import pool from "../../Database/database";
 import { verify } from "../../middleware/hash-password";
 import { signToken } from "../../middleware/jwt";
+// import jwt from "jsonwebtoken";
+//import Token from "../../models/token";
```
Comment (PrinceNiyonshuti) [inline src/Controllers/Authentication/login.controller.js]: remember to remove the unused or commented code block

File: src/Controllers/Authentication/login.controller.js [hunk 2]
```diff
@@ -3,51 +3,83 @@
...
+  if (email == "admin@gmail.com" && password == "password") {
+    const userData = {
+      name: "Admin User",
+      role: "admin",
+      email: email,
+      createdAt: new Date(),
+      id: 1,
+    };
+     const token = signToken(JSON.stringify(userData));
+    // const TokenInDatabase =await Token.create({token_content:token, userId:userData.id});
```
Comment (PrinceNiyonshuti) [inline src/Controllers/Authentication/login.controller.js]: also remember to remove the unused or commented code block

**Review by PrinceNiyonshuti — CHANGES_REQUESTED** [review]
Work on the commented codes , otherwise it looks good to me

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
Good work, though you have to resolve the conflicts in this branch

File: src/Controllers/Authentication/login.controller.js (hunk 1 above)
Comment (Hodal66) [reply src/Controllers/Authentication/login.controller.js]: Yeah Thanks Prince, Let me do It.

File: src/Controllers/Authentication/login.controller.js (hunk 2 above)
Comment (Hodal66) [reply src/Controllers/Authentication/login.controller.js]: Yeah Done!!!!, Thanks

**Comment by benshyaka** [comment]
Good work!

**Review by Xldivin — APPROVED** [review]

**Review by thepatrickniyo — APPROVED** [review]

**Review by PrinceNiyonshuti — APPROVED** [review]
looks good

**Review by tysepa — APPROVED** [review]

**Review by tysepa — APPROVED** [review]
Try to solve the conflicts but that is good work

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
Please resolve the conflicts

**Review by thepatrickniyo — APPROVED** [review]

**Pull request merged**